src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -295,7 +299,23 @@ watch(
                 </div>
                 <!-- eslint-disable vue/no-v-html-->
                 <div class="small text-wrap text-center" v-html="t('share_disclaimer')"></div>
-                <!-- eslint-enable vue/no-v-html-->
+                <div
+                    v-if="hasAnyLocalFile"
+                    class="d-flex flex-row gap-2 justify-content-center mt-2"
+                    data-cy="warn-share-local-file-container"
+                >
+                    <FontAwesomeIcon
+                        class="data-disclaimer-tooltip text-warning align-self-center"
+                        icon="circle-exclamation"
+                        data-cy="menu-external-disclaimer-icon-cloud"
+                    />
+                    <div
+                        class="small text-wrap text-center"
+                        data-cy="warn-share-local-file"
+                        v-html="t('warn_share_local_file')"
+                    ></div>
+                </div>
+                <!-- eslint-enable vue/no-v-html -->
```

Comment:
Might be better to move this eslint disabling rule to be set just after the line where `v-html` is used, here it is then disabled for part of the code where all rules should be applied.